Glue documentation review (#1456)
spryker/spryker-docs: docs/sdk/dev/value-resolvers.md

> # Spryker SDK value resolvers

Author: ```suggestion
---
title: Value resolvers
description: The Spryker SDK has several value resolvers described in this document.
template: concept-topic-template
---
# Spryker SDK value resolvers
```

description: The Spryker SDK has several value resolvers described in this document.
template: concept-topic-template
---

> # Spryker SDK value resolvers

Reviewer: as you already have the title in the first line, using one more H1 title is redundant

| NAMESPACE                | Used for the spryk tool to resolve namespaces for the tool. It is based on settings.                              |
| PC_SYSTEM                | Resolves the operating system like Linux, Mac, Mac (ARM).                                                       |
| PRIORITY_PATH            | Resolves the relative path for the tool entry point: `project` path, then, by default, the `sdk` path if it exists.       |
| REPORT_DIR               | Resolves the report file path.                                                                 |
| SDK_DIR                  | Resolves the path to the sdk folder.                             |
| CORE                     | Resolves the core level. This resolver is based on the resolved values of the `NAMESPACE` resolver.                   |
| STATIC                   | This resolver is used as a universal value resolver with additional settings. It returns the formatted value.                  |
| ORIGIN                   | This resolver is used as a universal value resolver with additional settings. It returns the original value.                   |

> Example of a placeholder with the `STATIC` value resolver and with the configuration. See [conventions](conventions.md#Placeholder) for details:

Author: ```suggestion
Example of a placeholder with the `STATIC` value resolver and with the configuration. See [conventions](/docs/sdk/conventions.html#Placeholder) for details:
```

> Example of a placeholder with the `STATIC` value resolver and with the configuration. See [conventions](/docs/sdk/conventions.html#Placeholder) for details:

Author: ```suggestion
Example of a placeholder with the `STATIC` value resolver and with the configuration. See [conventions](/docs/sdk/dev/conventions.html#Placeholder) for details:
```

> Example of a placeholder with the `STATIC` value resolver and with the configuration. See [conventions](/docs/sdk/dev/conventions.html#Placeholder) for details:

Author: ```suggestion
Example of a placeholder with the `STATIC` value resolver and with the configuration:
```
